### PR TITLE
Feature/faq and guidelines

### PR DESCRIPTION
**Pull request by volkansahn**
[Issue 45](https://github.com/kulkultech/open-source/issues/45) and [Issue 21](https://github.com/kulkultech/open-source/issues/21)

Created a document for Guidelines and FAQ for Github Commands (also hints).

### CODE REVIEW

**Comment by empeje** [comment]
Great job for your new PR @volkansahn !

However, I see that you didn't show your preview URL.

File: Frequently Asked Questions and Guidelines.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Frequently Asked Questions and Guidelines
```
Comment (empeje) [inline Frequently Asked Questions and Guidelines.md:1]: We have `faq.md` are you aware of that file? If not, do you think this document would more fit there?

Comment (volkansahn) [reply Frequently Asked Questions and Guidelines.md:1]: Hi, I haven't seen that faq.md document you are right that topic need to be in that document. I will append that document. Also as I didn't change anything on the webpage I don't need to get any preview, do I ?

Comment (empeje) [reply Frequently Asked Questions and Guidelines.md:1]: Hi @volkansahn, ah I think I misunderstood you.

Actually, we have a dedicated FAQ which available to view mdwiki (web page) as well, so a preview URL still needed.

**Comment by empeje** [comment]
Closed in lieu of #48

**Pull request closed without merging**